modules/axhal/src/platform/aarch64_phytium_pi/mod.rs

```diff
@@ -0,0 +1,73 @@
+use mp::cpu_hard_id_to_logic_id;
```

Comment:
The `cpu_hard_id_to_logic_id` API maybe is not suitable to be defined in `mp.rs`(which need to enabled under `SMP` features). I think it can be defined in `mod.rs`.